From foo@baz Mon Sep 14 14:34:20 PDT 2009
Date: Mon, 14 Sep 2009 14:34:20 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: hv: update the TODO file

Remove a few items that have already been resolved.
There are only a few checkpatch issues, they need to be resolved
by larger code logic changes that are not "simple" changes.

Cc: Hank Janssen <hjanssen@microsoft.com>
Cc: Haiyang Zhang <haiyangz@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/TODO |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/drivers/staging/hv/TODO
+++ b/drivers/staging/hv/TODO
@@ -1,8 +1,6 @@
 TODO:
-	- fix checkpatch warnings/errors
-	- fix sparse issues
+	- fix remaining checkpatch warnings and errors
 	- remove RingBuffer.c to us in-kernel ringbuffer functions instead.
-	- fix HANDLE usage to be "real" pointers
 	- audit the vmbus to verify it is working properly with the
 	  driver model
 	- see if the vmbus can be merged with the other virtual busses
